presto-main/src/test/java/com/facebook/presto/type/TestRowOperators.java

```diff
@@ -156,17 +156,31 @@ public void testIsDistinctFrom()
     }
 
     @Test
-    public void testRowEquality()
+    public void testRowComparison()
             throws Exception
     {
         assertFunction("row(TIMESTAMP '2001-01-02 03:04:05.321 +07:09', TIMESTAMP '2001-01-02 03:04:05.321 +07:10') = " +
                 "row(TIMESTAMP '2001-01-02 03:04:05.321 +07:09', TIMESTAMP '2001-01-02 03:04:05.321 +07:10')", BOOLEAN, true);
+        assertFunction("row(TIMESTAMP '2001-01-02 03:04:05.321 +07:09', TIMESTAMP '2001-01-02 03:04:05.321 +07:10') >= " +
+                "row(TIMESTAMP '2001-01-02 03:04:05.321 +07:09', TIMESTAMP '2001-01-02 03:04:05.321 +07:10')", BOOLEAN, true);
+        assertFunction("row(TIMESTAMP '2001-01-02 03:04:05.321 +07:09', TIMESTAMP '2001-01-02 03:04:05.321 +07:10') > " +
+                "row(TIMESTAMP '2001-01-02 03:04:05.321 +07:09', TIMESTAMP '2001-01-02 03:04:05.321 +07:10')", BOOLEAN, false);
+        assertFunction("row(TIMESTAMP '2001-01-02 03:04:05.321 +07:09', TIMESTAMP '2001-01-02 03:04:05.321 +07:10') > " +
+                "row(TIMESTAMP '2001-01-02 03:04:05.321 +07:09', TIMESTAMP '2000-01-02 03:04:05.321 +07:10')", BOOLEAN, true);
         assertFunction("row(1.0, row(TIMESTAMP '2001-01-02 03:04:05.321 +07:09', TIMESTAMP '2001-01-02 03:04:05.321 +07:10')) =" +
                 "row(1.0, row(TIMESTAMP '2001-01-02 03:04:05.321 +07:09', TIMESTAMP '2001-01-02 03:04:05.321 +07:10'))", BOOLEAN, true);
         assertFunction("row(1.0, 'kittens') = row(1.0, 'kittens')", BOOLEAN, true);
+        assertFunction("row(1.0, 'kittens') < row(1.0, 'kittens')", BOOLEAN, false);
+        assertFunction("row(1.0, 'kittens') < row(1.0, 'puppies')", BOOLEAN, true);
+        assertFunction("row(1.0, 'kittens') <= row(1.0, 'puppies')", BOOLEAN, true);
+        assertFunction("row(1.0, 'kittens') <= row(1.0, 'kittens')", BOOLEAN, true);
         assertFunction("row(1, 2.0) = row(1, 2.0)", BOOLEAN, true);
+        assertFunction("row(5, 2.0) > row(1, 2.0)", BOOLEAN, true);
+        assertFunction("row(5, 2.0) >= row(1, 3.0)", BOOLEAN, true);
+        assertFunction("row(5, 1.0) < row(1, 2.0)", BOOLEAN, false);
```

Comment:
For each operator (`<`, `<=`, `>`, `>=`), test less-than, equal-to, and greater-than.